test_main.py

```diff
@@ -37,3 +39,31 @@ def test_import():
     finally:
         os.environ.clear()
         os.environ.update(orig_env)
+
+def test_url_caching(empty_deployment):
+    publisher = kcidb.mq.URLListPublisher(
+        os.environ["PROJECT"],
```

Comment:
Uh, this one should be `GCP_PROJECT`, shouldn't it?

Reply:
Yeah it was just a demo layout to show u. I did changed it.